xds/internal/httpfilter/httpfilter.go

```diff
@@ -65,9 +65,6 @@ type ClientInterceptorBuilder interface {
 
 // ServerInterceptorBuilder constructs a Server Interceptor.  If this type is
 // implemented by a Filter, it is capable of working on a server.
-//
-// Server side filters are not currently supported, but this interface is
-// defined for clarity.
```

Comment:
Still mostly true, but removing this is fine.  Just a note for future reference: I doubt we'll want to define server interceptors in `iresolver` since servers and resolvers have nothing to do with each other.

Reply:
Ack.